bridge/config/config.go

```diff
@@ -115,6 +115,7 @@ type Protocol struct {
 	ShowEmbeds             bool       // discord
 	SkipTLSVerify          bool       // IRC, mattermost
 	StripNick              bool       // all protocols
+	SyncTopicChange        bool       // slack
```

Comment:
This was kinda discuss before, but should we use something more clear for `SyncTopicChange` and `ShowTopicChange`, since they apply to "purpose" too?

Maybe `SyncChannelMetaChanges` or `ShowChannelMetaChanges`? Kinda long though...